pkg/version/version.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/blang/semver"
+	log "github.com/sirupsen/logrus"
+)
+
+// providerVersion holds current provider version
+type providerVersion struct {
+	Version string `json:"version"`
+	Date    string `json:"date"`
+	// MinDriverVersion is minimum driver version the provider works with
+	MinDriverVersion string `json:"minDriverVersion"`
```

Comment:
maybe make a note about how this value will be used in the future?

Reply:
done!